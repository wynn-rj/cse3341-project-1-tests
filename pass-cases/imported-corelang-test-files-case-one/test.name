case one (Imported from corelan-test-files)
