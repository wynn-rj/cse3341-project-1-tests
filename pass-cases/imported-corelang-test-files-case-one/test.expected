program
  int x;
begin
  input x;
  case x of
    0:2
    else 4
  end;
  output x;
end
4
